core/controllers/contributor_dashboard.py

```diff
@@ -841,7 +856,9 @@ def get(self) -> None:
 
     @acl_decorators.can_manage_own_account
     def post(self) -> None:
-        """Handles POST requests."""
+        """Handles POST requests for updating the
+        user preferred translation language.
```

Comment:
It's ok to move more of this to the previous line (at least, what can be moved without going past.the 80 char limit.

Also, user --> user's